circle.yml

```diff
@@ -18,7 +18,7 @@ test:
     - npm run test-cov
 deployment:
   npm:
-    branch: master
+    tag: /[0-9]+.*/
```

Comment:
After discussed with Kumano-san, the regx can be `[0-9]+(\.[0-9]+){0,2}(-.*)?`(but please check carefully). 

### Expected match
1
0.3
0.4.5
0.4.5-beta
0.3-beta

### Expected non-match
0beta
0.1.1.1 (more than 2 dots)
abc
1.abc